### PR TITLE
feat: Update depth presets for Capri

### PR DESCRIPTION
**Pull request by knipec**
For depth presets:
- remove LandPKS
- add BLM monitoring standard
- make NRCS the default
- update tests
- create a db migration that will update any sites or projects with "LANDPKS" depth preset to "CUSTOM" depth preset with no depth intervals. 

### Related Issues
Backend part to address [#1541](https://github.com/techmatters/terraso-mobile-client/issues/1541)

### CODE REVIEW

File: terraso_backend/apps/soil_id/models/__init__.py [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 from .depth_dependent_soil_data import DepthDependentSoilData
 from .project_soil_settings import (
+    BLMStandardIntervalDefaults,
```
Comment (paulschreiber) [inline terraso_backend/apps/soil_id/models/__init__.py:19]:
```suggestion
    BLMIntervalDefaults,
```

File: terraso_backend/apps/soil_id/models/__init__.py [hunk 2]
```diff
@@ -31,8 +31,8 @@
     "ProjectSoilSettings",
     "ProjectDepthInterval",
     "SoilDataDepthInterval",
-    "LandPKSIntervalDefaults",
     "NRCSIntervalDefaults",
+    "BLMStandardIntervalDefaults",
```
Comment (paulschreiber) [inline terraso_backend/apps/soil_id/models/__init__.py:35]:
```suggestion
    "BLMIntervalDefaults",
```

File: terraso_backend/apps/soil_id/models/project_soil_settings.py [hunk 3]
```diff
@@ -21,21 +21,12 @@
 
 
 class DepthIntervalPreset(models.TextChoices):
-    LANDPKS = "LANDPKS"
     NRCS = "NRCS"
+    BLM_STANDARD = "BLM_STANDARD"
```
Comment (paulschreiber) [inline terraso_backend/apps/soil_id/models/project_soil_settings.py:25]:
```suggestion
    BLM = "BLM"
```

File: terraso_backend/apps/soil_id/models/project_soil_settings.py [hunk 4]
```diff
@@ -45,6 +36,14 @@ class DepthIntervalPreset(models.TextChoices):
     dict(depth_interval_start=100, depth_interval_end=200),
 ]
 
+BLMStandardIntervalDefaults = [
```
Comment (paulschreiber) [inline terraso_backend/apps/soil_id/models/project_soil_settings.py:39]:
```suggestion
BLMIntervalDefaults = [
```

File: terraso_backend/apps/soil_id/models/soil_data.py [hunk 5]
```diff
@@ -206,13 +206,13 @@ class Grazing(models.TextChoices):
     grazing_select = models.CharField(blank=True, null=True, choices=Grazing.choices)
 
     class SoilDataDepthIntervalPreset(models.TextChoices):
-        LANDPKS = "LANDPKS"
+        BLM_STANDARD = "BLM_STANDARD"
```
Comment (paulschreiber) [inline terraso_backend/apps/soil_id/models/soil_data.py:209]:
```suggestion
        BLM = "BLM"
```

File: terraso_backend/tests/graphql/mutations/test_soil_data.py [hunk 6]
```diff
@@ -562,7 +562,7 @@ def test_update_project_depth_intervals(client, project_manager, project):
     ]
 
 
-@pytest.mark.parametrize("preset", ["LANDPKS", "NRCS", "NONE"])
+@pytest.mark.parametrize("preset", ["NRCS", "BLM_STANDARD", "NONE"])
```
Comment (paulschreiber) [inline terraso_backend/tests/graphql/mutations/test_soil_data.py:565]:
```suggestion
@pytest.mark.parametrize("preset", ["NRCS", "BLM", "NONE"])
```

File: terraso_backend/tests/graphql/mutations/test_soil_data.py [hunk 7]
```diff
@@ -628,7 +628,7 @@ def test_update_project_soil_settings(client, user, project_manager, project):
 
     new_data = {
         "projectId": str(project.id),
-        "depthIntervalPreset": "NRCS",
+        "depthIntervalPreset": "BLM_STANDARD",
```
Comment (paulschreiber) [inline terraso_backend/tests/graphql/mutations/test_soil_data.py:631]:
```suggestion
        "depthIntervalPreset": "BLM",
```

File: terraso_backend/tests/graphql/mutations/test_soil_data.py [hunk 8]
```diff
@@ -666,7 +666,7 @@ def test_update_project_soil_settings(client, user, project_manager, project):
     assert payload == new_data
 
 
-@pytest.mark.parametrize("depth_interval_preset", ["LANDPKS", "NRCS", "CUSTOM"])
+@pytest.mark.parametrize("depth_interval_preset", ["NRCS", "BLM_STANDARD", "CUSTOM"])
```
Comment (paulschreiber) [inline terraso_backend/tests/graphql/mutations/test_soil_data.py:669]:
```suggestion
@pytest.mark.parametrize("depth_interval_preset", ["NRCS", "BLM", "CUSTOM"])
```

**Review by paulschreiber — APPROVED** [review]
Approved. Suggested a name change. (You'll need to regenerate the `.graphql` file as well.

**Review by paulschreiber — APPROVED** [review]

**Review by shrouxm — APPROVED** [review]
j'approve

**Pull request merged**